### ISSUE TITLE
tests bundle not excluded from production build?

### ISSUE THREAD

**Issue by NullVoxPopuli** [ef4/ember-auto-import#105]
I'm sure the title could be better, but, eh.

### Steps to Reproduce

Clone this repo: https://gitlab.com/NullVoxPopuli/emberclear/

run `cd packages/frontend; yarn build:production`

Get error:

```
Build failed.
Build Error (Bundler)

bundleForPath("ember-intl/test-support/-private/pick-last-locale.js") returned tests" but the only configured bundle names are app
```

### Environment

- Ember Version: 49e58e45604bd63ac1001cb491195c032c050053
- Ember CLI Version: 0f9d15d011fd9adb4c2a8db52e44bf5d74679631
- Ember Intl Version:  3.1.1
- Browser(s): n/a
- Node Version: 8.11.3

```
⬡ v8.11.3  (master) $ yarn why ember-auto-import
yarn why v1.7.0
[1/4] Why do we have the module "ember-auto-import"...?
[2/4] Initialising dependency graph...
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "ember-auto-import@1.2.10"
info Reasons this module exists
   - "ember-intl" depends on it
   - Hoisted from "ember-intl#ember-auto-import"
```

Related Issues:
https://github.com/ember-intl/ember-intl/issues/599

From @ef4 

![image](https://user-images.githubusercontent.com/199018/44283495-9103b480-a22c-11e8-9864-59cad4e2696f.png)

setting tests to true does indeed work. -- but obviously, I'd prefer to not have tests bundled in production mode.

the build takes forrreeevvveeerrrr though.

**Comment by ef4** [comment]
An annoying thing here is that this bug can only happen if ember-cli is trying to transpile the test-support files from ember-intl, despite the fact that you have disabled testing. Which is wasted work.

**Comment by NullVoxPopuli** [comment]
weirdness

**Comment by ef4** [comment]
This should be fixed in 1.2.11.
